### ISSUE TITLE
RTD warning about configuration file

### ISSUE THREAD

**Issue by nden** [spacetelescope/wfc3tools#81]
On ReadTheDocs I see this warning for wfc3tools:
```
Your project(s) "stistarg, wfc3tools, stsci-env" don't have a configuration
 file. Configuration files will soon be required by projects, and will no 
longer be optional. 
[Read our blog post to create one](https://blog.readthedocs.com/migrate-configuration-v2/) 
and ensure your project continues building successfully.
```

**Comment by nden** [comment]
Fixed by #84

**Comment by nden** [comment]
Fixed by #84
